build/rpm/main.go

```diff
@@ -37,8 +41,8 @@ type RpmSpec struct {
 	Name        string
 	Summary     string
 	MakeTarget  string
-	ProgramName string // kcn, kpn, ken, kscn, kbn
-	DaemonName  string // kcnd, kpnd, kend, kscn, kbn
+	ProgramName string // kcn, kpn, ken, kscn, kspn, ksen, kbn
+	DaemonName  string // kcnd, kpnd, kend, kscn, kspn, ksen, kbn
```

Comment:
```suggestion
	DaemonName  string // kcnd, kpnd, kend, kscnd, kspnd, ksend, kbnd
```
Probably the suffix `d` is needed?